cms/models_test.py

```diff
@@ -86,6 +89,37 @@ def test_notification_snippet():
     assert str(notification) == message_text
 
 
+def test_webinar_course():
+    """
+    Verify `course` property from the webinar page returns expected value
+    """
+    course = CourseFactory.create()
+    webinar_page = WebinarPageFactory.create(course=course)
+    assert webinar_page.course == course
+
+
+def test_upcoming_webinar_date_time():
+    """
+    Verify `date` and `time` property for the upcoming webinar to be present
+    """
+    webinar = WebinarPageFactory.create()
+    assert webinar.category == UPCOMING_WEBINAR
+    assert webinar.date
+    assert webinar.time

```

Comment:
How about we create a webinar with a specific date and time and then verify that date and time?